exampleSite/content/index/contact.md

```diff
@@ -17,6 +17,10 @@ email = "mail@example.com"
 button = "Send Button" # defaults to theme default
 #netlify = false
 
+# Optional google captcha
+#[params.contact.captcha]
```

Comment:
Could probably be simplified to just:
```
#[captcha]
#  sitekey = ""
```
No need to have the more complex format.

Reply:
Or [recaptcha] to make it clear, that it's a setting for google's recaptcha.